CharlieBackend.AdminPanel/Models/EventOccurence/EventOccurenceViewModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿using CharlieBackend.Core.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CharlieBackend.AdminPanel.Models.EventOccurence
+{
+    public class EventOccurenceViewModel
```

Comment:
Why are you adding `EventOccurenceViewModel`? Where is it used?

Reply:
I didnt remove it after merging your task. 9addd8f7c3c858f432346be32c39219df9043ca0
Removed.